apis/script/v1/script_types.go

```diff
@@ -42,14 +42,18 @@ type ScriptSpec struct {
 
 // Repository represents VCS repo, currently we're habdling Git only
 type Repository struct {
-	// VCS repository type
+	// Type_ repository type
 	Type_ string `json:"type"`
-	// uri of content file or git directory
+	// Uri of content file or git directory
 	Uri string `json:"uri"`
 	// branch/tag name for checkout
 	Branch string `json:"branch"`
 	// if needed we can checkout particular path (dir or file) in case of BIG/mono repositories
 	Path string `json:"path,omitempty"`
+	// git auth username for private repositories
```

Comment:
I'm a little afraid about obfuscation of this data, shouldn't we simply pass reference to a secret ? 
what do you think guys?

Reply:
just looked at create - it need to be here :) ignore my last comment - mapper need to be changed to covert it - and probably is already :)